hermeto/interface/cli.py

```diff
@@ -174,6 +174,11 @@ def main(  # noqa: D103; docstring becomes part of --help message
         case_sensitive=False,
         help="Set log level.",
     ),
+    mode: Mode = typer.Option(
+        Mode.STRICT,
+        "--mode",
+        help="Set input expectations and requirements validation mode.",
```

Comment:
This is a NOP at the moment, can we mark it as such in the CLI help? Putting something like `(currently a NOP)` to the help output?

Reply:
That would need some hook to not forget to remove it once it is no longer a stub.

Reply:
Fair enough, just docs it is then.